features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesView.kt

```diff
@@ -512,15 +512,15 @@ private fun RoomAvatarAndNameRow(
 private fun CantSendMessageBanner() {
     Row(
         modifier = Modifier
-                .fillMaxWidth()
-                .background(MaterialTheme.colorScheme.secondary)
-                .padding(16.dp),
+            .fillMaxWidth()
+            .background(ElementTheme.colors.bgSubtleSecondary)
+            .padding(16.dp),
         verticalAlignment = Alignment.CenterVertically,
         horizontalArrangement = Arrangement.Center
     ) {
         Text(
             text = stringResource(id = R.string.screen_room_timeline_no_permission_to_post),
-            color = MaterialTheme.colorScheme.onSecondary,
+            color = ElementTheme.colors.textSecondary,
```

Comment:
Using Semantic colors for this banner, waiting for design input that can be handled later.